io.neovim.nvim.tool.tmux.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.neovim.nvim.tool.tmux
+branch: stable
+runtime: io.neovim.nvim
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: libevent
+    sources:
+      - type: archive
+        url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+        sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=${FLATPAK_DEST} --enable-shared
+      - make
+      - make install
+  - name: libutempter
+    sources:
+      - type: archive
+        url: https://github.com/altlinux/libutempter/archive/refs/tags/1.2.1-alt1.tar.gz 
+        sha256: 258d8bc4a42d2cbe4df6415912ab842574899adddbfc89d14ab56b57bb81158b 
+    buildsystem: simple
+    build-commands:
+      - make -C libutempter DESTDIR=${FLATPAK_DEST} libdir=/lib libexecdir=/lib includedir=/include
+      - make -C libutempter DESTDIR=${FLATPAK_DEST} libdir=/lib libexecdir=/lib includedir=/include install
+  - name: tmux
+    sources:
+      - type: archive
+        url: https://github.com/tmux/tmux/releases/download/3.3a/tmux-3.3a.tar.gz
+        sha256: e4fd347843bd0772c4f48d6dde625b0b109b7a380ff15db21e97c11a4dcdf93f
+    buildsystem: simple
+    build-commands:
+      - ./configure LIBRARY_PATH=${FLATPAK_DEST}/lib:$LIBRARY_PATH PKG_CONFIG_PATH=${FLATPAK_DEST}/lib/pkgconfig LDFLAGS=-L${FLATPAK_DEST}/lib CFLAGS=-I${FLATPAK_DEST}/include --prefix=${FLATPAK_DEST} --enable-utempter
```

Comment:
this should be buildsystem: autotools instead.  With the proper options.

Reply:
Unfortunately, I cannot get this one to compile with autotools. It keeps failing to find libevent. I'll maybe try again in a few days but let me know if you have any suggestions.

Reply:
you need `prepend-pkg-config-path` in the build options.

Reply:
Thank you for your patience and reviewing these. I believe it's now fixed here https://github.com/flathub/flathub/pull/4119/commits/c55d968f1cd954dee52d4439b7c6da20a2108e2a